packages/permissions/src/components/authorized/authorized.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import invariant from 'tiny-invariant';
+import { TPermissions, TPermissionName } from '../../types';
```

Comment:
We actually can't not have the same types in `types.ts` and inline them for prop-types as they won't match. At least TS complained.